.circleci/config.yml

```diff
@@ -15,11 +15,12 @@ aliases:
         name: db
   - &frontend_base
     docker:
-      - image: registry.opensuse.org/obs/server/unstable/container/sle12/sp3/images/openbuildservice/frontend-base:latest
+      - image: registry.opensuse.org/home/coolo/branches/obs/server/unstable/container/sle12/sp3/containers/openbuildservice/frontend-base:latest
```

Comment:
Can you drop this? Your SR is merged :-)

Reply:
done